Dockerfile

```diff
@@ -15,7 +15,7 @@ ENTRYPOINT ["bundle", "exec"]
 CMD ["rails db:migrate && rails server"]
 
 # patches
-RUN apk add --no-cache postgresql14=14.5-r0 libtasn1=4.18.0-r1 
+RUN apk add --no-cache libtasn1=4.18.0-r1 openssl=1.1.1t-r0 
```

Comment:
The postgres patch version is no longer required ?

Reply:
Yeah, so it tries to grab a newer version by default now and we get this error:

```
#7 0.834 ERROR: unable to select packages:
#7 0.869   postgresql14-14.7-r0:
#7 0.869     breaks: world[postgresql14=14.5-r0]
```

So I removed the pinned version to let it go to 14.7